flink-table/flink-sql-gateway-api/src/test/java/org/apache/flink/table/gateway/api/utils/MockedSqlGatewayEndpointFactory.java

```diff
@@ -41,12 +40,14 @@ public class MockedSqlGatewayEndpointFactory implements SqlGatewayEndpointFactor
 
     @Override
     public SqlGatewayEndpoint createSqlGatewayEndpoint(Context context) {
-        ReadableConfig config = context.getConfiguration();
-        SqlGatewayEndpointFactoryUtils.createSqlGatewayEndpointFactoryHelper(this, context)
-                .validate();
+        SqlGatewayEndpointFactoryUtils.EndpointFactoryHelper helper =
+                SqlGatewayEndpointFactoryUtils.createEndpointFactoryHelper(this, context);
+        helper.validate();
 
         return new MockedSqlGatewayEndpoint(
-                config.get(HOST), config.get(PORT), config.getOptional(DESCRIPTION).orElse(null));
+                helper.getOptions().get(HOST),
+                helper.getOptions().get(PORT),
+                helper.getOptions().getOptional(DESCRIPTION).orElse(null));
```

Comment:
minor: should be `helper.getOptions().get(DESCRIPTION)` to use default value if not present.